### ISSUE TITLE
Support for the experimental syntax 'classPrivateProperties' isn't currently enabled (15:23):

### ISSUE THREAD

**Issue by zehawki** [Steve-Junior/tawkto-react#28]
./node_modules/tawkto-react/src/index.js
SyntaxError: ...\node_modules\tawkto-react\src\index.js: Support for the experimental syntax 'classPrivateProperties' isn't currently enabled (15:23):

  13 | class TawkTo {
  14 |   constructor(propertyId, widgetId, zIndex = null) {
> 15 |     this._Tawk = this.#init(propertyId, widgetId, zIndex);
     |                       ^
  16 |   }
  17 |
  18 |   #init(propertyId, widgetId, zIndex) {

Add @babel/plugin-proposal-class-properties (https://git.io/vb4SL) to the 'plugins' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-class-properties (https://git.io/vb4yQ) to the 'plugins' section to enable parsing.

**Comment by ghost** [comment]
Hello @zehawki Can you help with how to reproduce this error?

**Comment by zehawki** [comment]
I didnt have to do anything much... 

Just did a `yarn add "tawkto-react"`, then import. This is on CRA.

**Comment by rares-lupascu** [comment]
is this related?
![image](https://user-images.githubusercontent.com/3641866/149509256-bc6b920b-57a2-4360-8402-1efc5b3db86f.png)

**Comment by zehawki** [comment]
Yup. You may see that's exactly the error at line (L15) in what I reported above.

**Comment by rares-lupascu** [comment]
```
yarn add -D @babel/plugin-proposal-class-properties
```
```
module.exports = {
    module: {
        rules: [
            {
                test: path.join(__dirname, '.'),
                exclude: /(node_modules)/,
                loader: 'babel-loader',
                options: {
                    presets: [
                        // '@babel/preset-env',
                        // '@babel/react',
                        {
                            plugins: [
                                '@babel/plugin-proposal-class-properties',
                            ],
                        },
                    ],
                },
            },
            {
                test: /\.svg$/,
                use: ['@svgr/webpack'],
            },
        ],
    },
};
```
this should get it working but I think this needs to be in the official docs

**Comment by zehawki** [comment]
Like I said, I use Create React App, so cant do that. Also not going to enable anything experimental on production systems.

**Comment by ghost** [comment]
Yes, I agree. Not many people will be okay placing an experimental feature on production. I have rolled out an update to fix it v3.0.1

**Comment by zehawki** [comment]
Awesome, I'm off to try it out again.
